include/fmt/color.h

```diff
@@ -192,6 +191,13 @@ enum class color : uint32_t {
   yellow_green            = 0x9ACD32  // rgb(154,205,50)
 };  // enum class color
 
+enum class emphasis : uint32_t {
+  bold = 1 << 1,
+  italic = 1 << 3,
```

Comment:
Why gaps in the bit pattern?

Reply:
The numbers were originally for the escape code, but since many different emphasis can be ORed together, I used this, but I didn't change the actual code. I guess I can remove the bits haha :)

Reply:
Yeah, let's remove the gaps.